### ISSUE TITLE
handbook headings 

### ISSUE THREAD

**Issue by ynnig** [doi-foundation/doi-website#33]
From Wouter, DISSco:

**Comment by ynnig** [comment]
![factsheets doi](https://user-images.githubusercontent.com/6958740/221156867-f612a5c3-7f64-4ddb-82a0-46eaa84745dc.jpg)
The default page design needs reducing, perhaps normalising on the Handbook page. @cazinc how did you do that and can we easily toggle between two page designs (circle-on/circle-off) in the frontmatter?

**Comment by JonathanClarkDOIF** [comment]
Think there are two issues here. I realise now that the Factsheets were simply ported without much thought on how they look. I checked a few of them and there’s always an introduction part that could easily be put in the block opposite the circle. I actually like the format and I’ll take an action to update them all. The other issue is if we need a simple page format for some content types without the circle and I totally agree here - definitely for some of the documentation I have yet to migrate - I don’t want to have to always create PDFs for this.

**Comment by ynnig** [comment]
Agreed - we just need to be able to toggle the circle on/off really and decide per page

On 24 Feb 2023, at 16:23, JonathanClarkDOIF ***@***.***> wrote:


Think there are two issues here. I realise now that the Factsheets were simply ported without much thought on how they look. I checked a few of them and there’s always an introduction part that could easily be put in the block opposite the circle. I actually like the format and I’ll take an action to update them all. The other issue is if we need a simple page format for some content types without the circle and I totally agree here - definitely for some of the documentation I have yet to migrate - I don’t want to have to always create PDFs for this.

**Comment by cazinc** [comment]
Hello, 

There are various options here:

1. Factsheets (or any set of pages) could be made to use the circle-off template automatically (like the Handbook);
2. We could enable a way to allow content editors to pick and choose per page; 
3. We could force all third level or lower pages to use the circle-off template. 

Option 1 is no problem, it would just be an an extension of how things work for the Handbook, and it has the advantage of being automated for only specific sets of pages. Option 2 is the most flexible, but used by itself could end up causing some inconsistency (it's easy to forget use it on 1 out of 10 pages for example, or an editor may not think about the pros/cons of using it on a specific page). Option 3 could cause problems as perhaps some third level pages would benefit from the larger header? 

I would suggest option 1, and of course we can apply this to any particular area of the site if need be. The basis would be a directory, like /the-identifier/resources/*factsheets*.

Another route could be options 1 and 2, i.e. some areas automatically use the circle-off, but editors can also use it on any given page via a frontmatter setting. 

Let me know your pref. 

We can also tweak the circle-off design any way desired, e.g. less spacing. It should be possible to do this in a way that is clear. Or we could work in a line or something - maybe a smaller circle element even - for consistency, clarity, etc.
